pkg/lib/operation/template/local/repository/list/operation.go

```diff
@@ -0,0 +1,36 @@
+package list
+
+import (
+	"github.com/keboola/keboola-as-code/internal/pkg/log"
+	"github.com/keboola/keboola-as-code/internal/pkg/template/repository"
+)
+
+type dependencies interface {
+	Logger() log.Logger
+}
+
+func Run(repo *repository.Repository, d dependencies) (err error) {
+	logger := d.Logger()
```

Comment:
`logger` ma metodu `InfoWriter()`, bolo by to asi trocha lepsie, hlavne by sa ten writer dal v pripade potreby vymenit za nejaky iny.

Reply:
Mozes tam pridat metodu `Writef`, ktora bude volat `Sprintf` + `Write`.

Reply:
ok, upraveno 👍